core/modules/mod.rs

```diff
@@ -725,7 +708,6 @@ pub(crate) struct ModuleRequest {
 pub(crate) struct ModuleInfo {
   #[allow(unused)]
   pub id: ModuleId,
-  // Used in "bindings.rs" for "import.meta.main" property value.
```

Comment:
Why remove this comment?

Reply:
It's no longer used there... we use the main_module_id property in ModuleMap now.

Reply:
Okay, so why keep this field?

Reply:
We use it as part of tests 😢